go.mod

```diff
@@ -4,8 +4,12 @@ require (
 	github.com/beorn7/perks v1.0.0
 	github.com/go-logfmt/logfmt v0.4.0 // indirect
 	github.com/golang/protobuf v1.3.1
+	github.com/json-iterator/go v1.1.6
+	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
+	github.com/modern-go/reflect2 v1.0.1 // indirect
 	github.com/prometheus/client_model v0.0.0-20190129233127-fd36f4220a90
-	github.com/prometheus/common v0.4.0
-	github.com/prometheus/procfs v0.0.0-20190507164030-5867b95ac084
+	github.com/prometheus/common v0.4.1-0.20190516160040-17f5ca174818
```

Comment:
This is an untagged version of `prometheus/common`.
- Is that really required?
- If yes, could we just ask @brian-brazil to tag a version in `prometheus/common`? It is very confusing for users of `client_golang` if they have an untagged dependency in an otherwise tagged repository.

Reply:
For the tests to pass this requires https://github.com/prometheus/common/pull/193 which is on master (otherwise untagged).

Reply:
I see. @brian-brazil I guess we could just tag a 0.4.1 in prometheus/common?